lib/widgets/qr_codes.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:get/get_navigation/get_navigation.dart';
+import 'package:get/state_manager.dart';
+import 'package:mi_utem/models/permiso_covid.dart';
+import 'package:mi_utem/screens/permiso_covid_screen.dart';
+import 'package:mi_utem/services/permisos_covid_service.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class QRCodes extends StatefulWidget {
+  const QRCodes({Key? key}) : super(key: key);
+
+  @override
+  State<QRCodes> createState() => _QRCodesState();
+}
+
+class _QRCodesState extends State<QRCodes> {
+  late Future<List<PermisoCovid>> permisos;
+
+  @override
+  initState() {
+    super.initState();
+
+    permisos = PermisosCovidService.getPermisos();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: 8.0, bottom: 30.0),
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(left: 30, right: 30),
+            child: Row(
+              children: [
+                Text(
+                  "PERMISOS ACTIVOS",
+                  style: TextStyle(
+                    color: Color(0xFF363636),
+                    fontWeight: FontWeight.w700,
+                  ),
+                ),
+                Text(
+                  "Ver Todos",
+                  style: TextStyle(
+                    color: Color(0xFF009D9B),
+                    fontWeight: FontWeight.w700,
+                  ),
+                ),
+              ],
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            ),
+          ),
+          SizedBox(
+            height: 175,
+            child: FutureBuilder(
+              future: permisos,
+              builder: (BuildContext context, AsyncSnapshot snapshot) {
+                if (snapshot.hasError) {
+                  return Text('${snapshot.error}');
+                }
+                if (!snapshot.hasData) {
+                  return const CircularProgressIndicator();
+                }
+
+                if (snapshot.data.length == 0) {
+                  return Text('No hay permisos COVID');
+                }
+
+                return ListView.builder(
```

Comment:
Cuando haces un ListView tienes que poner sí o sí el `itemCount` para indicarle cuántos ítems tendrá la lista, además falta establecer que la lista será horizontal para que el scroll quede bien, además de un padding para que quede alineado con los botones de abajo

```suggestion
                return ListView.builder(
                  scrollDirection: Axis.horizontal,
                  padding: EdgeInsets.symmetric(horizontal: 20),
                  itemCount: snapshot.data.length,
                  itemBuilder: (context, index) =>
                        QRCard(permiso: snapshot.data[index],
                  ),
                );
                
```